scipio/src/sys/mod.rs

```diff
@@ -114,7 +120,7 @@ pub(crate) enum SourceType {
     Close,
     LinkRings(bool),
     Statx(CString, Box<RefCell<libc::statx>>),
-    Timeout(bool),
+    Timeout(TimeoutStatus),
```

Comment:
I am about to push a change that will conflict heavily with this, and rewrites most of uring. (see https://github.com/DataDog/scipio/issues/52 - > the problem turned out to be quite nasty).

This is still a good change, but we'll conflict and what I am doing is a bit more delicate, so I'd like to defer this

Reply:
Note that this is an internal type. The change is still good, but less crucial.